python/identifiers.cpp

```diff
@@ -50,9 +50,14 @@ void register_identifiers(py::module& m) {
              "Construct a cell_local_label identifier with arguments:\n"
              "  label:  The identifier of a group of one or more items on a cell.\n"
              "  policy: The policy for selecting one of possibly multiple items associated with the label.\n")
-        .def(py::init([](py::tuple t) {
-               if (py::len(t)!=2) throw std::runtime_error("tuple length != 2");
-               return arb::cell_local_label_type{t[0].cast<arb::cell_tag_type>(), t[1].cast<arb::lid_selection_policy>()};
+        .def(py::init([](const std::tuple<arb::cell_tag_type, arb::lid_selection_policy>& t) {
+               return arb::cell_local_label_type{std::get<arb::cell_tag_type>(t), std::get<arb::lid_selection_policy>(t)};
+             }),
+             "Construct a cell_local_label identifier with tuple argument (label, policy):\n"
+             "  label:  The identifier of a group of one or more items on a cell.\n"
+             "  policy: The policy for selecting one of possibly multiple items associated with the label.\n")
+        .def(py::init([](const std::pair<arb::cell_tag_type, arb::lid_selection_policy>& t) {
+               return arb::cell_local_label_type{std::get<arb::cell_tag_type>(t), std::get<arb::lid_selection_policy>(t)};
```

Comment:
do we really need both `tuple` and `pair`?

Reply:
Probably no ... but what's the harm?

Reply:
nothing really, was just wondering